go/libkb/post.go

```diff
@@ -215,12 +215,13 @@ func CheckInvitationCode(code string) error {
 	return err
 }
 
-func GetInvitationCode() (string, error) {
+func GetInvitationCode(g *GlobalContext) (string, error) {
 	arg := APIArg{
 		Endpoint:    "invitation_bypass_request",
 		NeedSession: false,
+		NetContext:  g.GetNetContext(),
```

Comment:
this is hack i'd prefer not to use if we can avoid it.  can you pass both through?

```
GetInvitationCode(net context.Context, g *GlobalContext)
```